modules/planning/lattice/trajectory_generation/backup_trajectory_generator.cc

```diff
@@ -22,9 +22,10 @@ namespace apollo {
 namespace planning {
 
 using apollo::common::PathPoint;
+using State = std::array<double, 3>;
```

Comment:
recommend using a less generic name

Reply:
I think it is fine since it will be only used in this source file.